packages/core/src/render3/instructions.ts

```diff
@@ -1465,6 +1464,31 @@ function instantiateAllDirectives(tView: TView, viewData: LViewData, previousOrP
   }
 }
 
+function invokeDirectivesHostBindings(
+    tView: TView, viewData: LViewData, previousOrParentTNode: TNode) {
+  const start = previousOrParentTNode.flags >> TNodeFlags.DirectiveStartingIndexShift;
+  const end = start + (previousOrParentTNode.flags & TNodeFlags.DirectiveCountMask);
+  const expando = tView.expandoInstructions !;
+  for (let i = start; i < end; i++) {
+    const def = tView.data[i] as DirectiveDef<any>;
+    const directive = viewData[i];
+    if (def.hostBindings) {
+      const before = expando.length;
```

Comment:
Nit: more descriptive var name

```suggestion
      const previousExpandoLength = expando.length;
```